src/lib/firebase.ts

```diff
@@ -147,6 +158,47 @@ export class Firebase {
     this.groupOnDisconnect = null;
   }
 
+  //
+  // Firestore
+  //
+
+  public getPublicUrlFromStore(storePath: string): Promise<any> {
+    const ref = this.firestore().ref(storePath);
+    // Get the download URL - returns a url with an authentication token for the current session
+    return ref.getDownloadURL().then((url) => {
```

Comment:
Not sure this `then` statement is necessary, `ref.getDownloadURL()` should return correctly.